frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -105,6 +105,18 @@ export class StorageClassDropdownInner extends React.Component<
     }
   }
 
+  getLabelClassName = () => {
+    const classNames: string[] = [];
+    this.props.hideClassName
```

Comment:
Not new, but this is a confusing prop name. What is this for?

Reply:
it seems thats the prop is being used in places to add class names to the label. i am still keeping the changes for the required prop as it is well recognized .